server/controllers/assess/submittedApplicationsController.ts

```diff
@@ -17,13 +19,55 @@ export default class SubmittedApplicationsController {
     }
   }
 
+  private getSubmittedTimelineEvent(name: string, submittedAt: string): UiTimelineEvent {
+    return {
+      label: { text: 'Application submitted' },
+      byline: {
+        text: name,
+      },
+      datetime: {
+        timestamp: submittedAt,
+        date: DateFormats.isoDateTimeToUIDateTime(submittedAt),
+      },
+      description: {
+        text: 'The application was received by an assessor.',
+      },
+    }
+  }
+
+  private getStatusTimelineEvents(statusUpdates: Array<Cas2StatusUpdate>): Array<UiTimelineEvent> {
+    return statusUpdates
+      .sort((a, b) => Number(DateFormats.isoToDateObj(b.updatedAt)) - Number(DateFormats.isoToDateObj(a.updatedAt)))
+      .map(statusUpdate => {
+        return {
+          label: { text: statusUpdate.label },
+          byline: {
+            text: statusUpdate.updatedBy.username,
```

Comment:
I may have missed it elsewhere, but is it worth noting that this will be changing to a human name like "Roger Smith" once the API supplies this in the `ExternalUser`?